drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -1065,14 +1065,10 @@ Ref<Image> RasterizerSceneGLES3::sky_bake_panorama(RID p_sky, float p_energy, bo
 	texture.target = GL_TEXTURE_2D;
 	texture.active = true;
 	texture.tex_id = rad_tex;
-	texture.is_render_target = true;
+	texture.is_render_target = false;
```

Comment:
If this is true, you cannot free the texture:

https://github.com/godotengine/godot/blob/60913170c696aa8714f009b84cf7106ed9d730a0/drivers/gles3/storage/texture_storage.cpp#L763

Reply:
This is a little hack to avoid the server from holding on to a cached copy of the texture in `texture_2d_get()`. That's why we set to to true, and then set it to false directly below before freeing

Reply:
Ok, I suspected that – do you have a suggestion for how to work around the issue that the free fails (and leaks) the texture when `is_render_target` is true? Perhaps the `texture` needs a `skip_cache` boolean option?

Reply:
I have an alternative which retains the hack and fixes the error 👍🏻